packages/bulk-edit/src/browser/bulk-edit-tree/in-memory-text-resource.ts

```diff
@@ -0,0 +1,40 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
```

Comment:
should it be SAP?

Reply:
I copied it (with a little change) from packages\search-in-workspace\src\browser\in-memory-text-resource.ts so that's why I left it like that. Should I change it?

Reply:
Fixed. thanks